### PR TITLE
chore: updated changelogs for hosted ui

### PR DESCRIPTION
**Pull request by haverchuck**
*Issue #, if available:*
Updated changelogs for hosted ui

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_auth_cognito/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ## 0.0.2-dev.1 (2021-01-25)
 
+### Features
+- feature: Amplify Hosted UI (#309)
```
Comment (Amplifiyer) [inline packages/amplify_auth_cognito/CHANGELOG.md:4]:
```suggestion
- feature: Amplify Auth Hosted UI (#309)
```

File: packages/amplify_auth_plugin_interface/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 ## 0.0.2-dev.1 (2021-01-25)
 
+### Features
+- feature: Amplify Hosted UI (#309)
```
Comment (Amplifiyer) [inline packages/amplify_auth_plugin_interface/CHANGELOG.md:4]:
```suggestion
- feature: Amplify Auth Hosted UI (#309)
```

**Review by jamesonwilliams — COMMENTED** [review]
Idle thought -- _we_ know what the hosted UI is. However, a newcomer may see this and not have any idea. A brief sentence describing what it does may be useful.

I think the key bullet here is really around third-party sign-in. Signing in Google, Facebook, and Apple is a _huge_ new capability. We should sell that customer scenario, here!

File: packages/amplify_flutter/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 ## 0.0.1-dev.7 (TBD)
```
Comment (Amplifiyer) [inline packages/amplify_flutter/CHANGELOG.md:1]: I'll fix that in a different PR.

File: packages/amplify_flutter/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
...
 
+### Features
+- feature: Amplify Hosted UI (#309)
```
Comment (Amplifiyer) [inline packages/amplify_flutter/CHANGELOG.md:4]:
```suggestion
- feature: Amplify Auth Hosted UI (#309)
```

**Review by Amplifiyer — APPROVED** [review]
Looks good. Small suggestion to add the word `Auth` in it

**Pull request merged**